src/views/htmlcontent/src/js/constants.ts

```diff
@@ -1,19 +1,24 @@
 /** Results Pane Labels */
-export const maximizeLabel = 'Maximize';
-export const restoreLabel = 'Restore';
-export const saveCSVLabel = 'Save as CSV';
-export const saveJSONLabel = 'Save as JSON';
-export const saveExcelLabel = 'Save as Excel';
-export const resultPaneLabel = 'Results';
-export const selectAll = 'Select all';
-export const copyLabel = 'Copy';
-export const copyWithHeadersLabel = 'Copy with Headers';
+export let maximizeLabel = 'Maximize';
```

Comment:
Please doc that all new strings added here must be added to the xliff. Otherwise people may miss this